backend/src/main/resources/db/migration/V1.23.0__NOTIF-190_cronjob_timestamp.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE cronjob_run (
+    id UUID NOT NULL,
+    last_run TIMESTAMP
+) WITH (OIDS=FALSE);
+
+INSERT INTO cronjob_run (id, last_run) VALUES
+('3db0955b-751b-48cd-b531-c1d81596d133', '-infinity');
```

Comment:
If the UUID purpose is to prevent multiple rows in the `cronjob_run` table, we already have something similar based on Postgres constraints with the application status. See [V1.20.0__NOTIF-226_maintenance_mode.sql](https://github.com/RedHatInsights/notifications-backend/blob/master/backend/src/main/resources/db/migration/V1.20.0__NOTIF-226_maintenance_mode.sql) and [CurrentStatus.java](https://github.com/RedHatInsights/notifications-backend/blob/master/backend/src/main/java/com/redhat/cloud/notifications/models/CurrentStatus.java#L34).

Reply:
the UUID is the Generated PK of the CronJob class.
I would say it doesn't need to exist somewhere as we are initializing it in the flyway script,
then we always read the entry: 

        final CronJobRun lastCronJobRun = emailAggregationResources.getLastCronJobRun();
        emailAggregationResources.updateLastCronJobRun(lastCronJobRun.getId(), now);

and always update the same entry over and over again.

Reply:
-Infinity is the oldest possible datetime entry in Postgres: https://stackoverflow.com/questions/19686635/earliest-timestamp-supported-in-postgresql

Reply:
The latest change from Instant to LocalDate has broken the code completely.

I will make it a draft now and work on it when I am back from vacation.